doc/api/next_api_changes/behaviour.rst

```diff
@@ -41,3 +41,12 @@ did nothing, when passed an unsupported value. It now raises a ``ValueError``.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ``backend_pgf.LatexManager.latex`` is now created with ``encoding="utf-8"``, so
 its ``stdin``, ``stdout``, and ``stderr`` attributes are utf8-encoded.
+
+``pyplot.xticks()`` and ``pyplot.yticks()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, passing labels without passing the ticks to either `.pyplot.xticks`
+and `.pyplot.yticks` would result in
+
+    TypeError: object of type 'NoneType' has no len()
+
+It now raises a ``ValueError`` with a proper description of the error.
```

Comment:
typeerror

Reply:
Damn, this is what happens when I rush things. I'm fixing this typo and squashing the commits. Sorry for this.